app/(dashboard)/mailboxes/[mailbox_slug]/settings/team/teamSetting.tsx

```diff
@@ -31,7 +31,6 @@ const TeamSetting = ({ mailboxSlug }: TeamSettingProps) => {
     <SectionWrapper
       title="Manage Team Members"
       description="Add and organize team members for efficient ticket assignment"
```

Comment:
```suggestion
      description="Add and organize team members for efficient ticket assignment"
      fullWidth
```

(it's needed on this one)